### DISCUSSION TITLE
Ecosystem Mapping projects

### DISCUSSION THREAD

**Discussion by catherinet1** [life-itself/community#506] [Research]
When we are doing work to map an emerging ecosystem of radical, alternative social change actors (e.g. https://lifeitself.org/ecosystem), we want to know what other similar efforts are out there, so that we can learn from and build on collective experience and thus work more effectively/coherently as a whole. e.g. we want to see what others have already done so that we understand better how our project can add value rather than duplicating effort.

List here...

| Name  | URL | Description | Visualisations/Views of data |
| ------------- | ------------- | ------------- | ------------- |
| Gaianet  | https://www.gaianet.earth/  | a worldwide virtual network where New Earth professionals, project leaders, experts, elders and investors connect and unite. currently consists of 600+ mission-aligned projects and 480+ trusted professionals (all personally invited) | [Gallery view of physical projects](https://airtable.com/shrONugiPQlE1onx3). Different views and filters [here](https://www.catalist.network/trove/gaianet?view=Graph&filtered=yes). |
| ...  | ...  | ... |

**Comment by rufuspollock** [comment]
@catherinet1 we had this old issue https://github.com/life-itself/ecosystem/issues/14

BTW: i've long planned to turn into a post / page on the website e.g. lifeitself.org/ecosystem/related or similar ...  - maybe now is the time!

PS: IMO screenshots are super helpful and you can just copy and paste images into discussion here.

PPS: hence a table layout is probably less useful for now than a headings model with bullets for attributes e.g.

```md
### Gaianet

- 🔗 https://gaianet.earth/

A worldwide virtual network where New Earth professionals, project leaders, experts, elders and investors connect and unite. currently consists of 600+ mission-aligned projects and 480+ trusted professionals (all personally invited)

#### Views of the data

[Gallery view of physical projects](https://airtable.com/shrONugiPQlE1onx3). Different views and filters [here](https://www.catalist.network/trove/gaianet?view=Graph&filtered=yes).
```
